src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -354,6 +354,26 @@ private static void EnsureScopedRefAttributeExists(CSharpCompilation compilation
             }
         }
 
+        internal static void EnsureRequiresLocationAttributeExists(CSharpCompilation? compilation, ImmutableArray<ParameterSymbol> parameters, BindingDiagnosticBag? diagnostics, bool modifyCompilation, PEModuleBuilder? moduleBuilder)
```

Comment:
>EnsureRequiresLocationAttributeExists [](http://example.com/codeflow?start=29&length=37)

I think we should merge what this function is doing into ```EnsureIsReadOnlyAttributeExists``` and rename it to "EnsureRefKindAttributesExists"